lib/mws/customer_information/client.rb

```diff
@@ -18,7 +18,8 @@ module CustomerInformation
     # @note In addition to registering for Amazon MWS, you must request
     #   authorization to use the Customer Information API.
     class Client < ::Peddler::Client
-      path '/CustomerInformation/2014-03-01'
+      version '2014-03-01'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.